src/views/product-landing/schema.ts

```diff
@@ -0,0 +1,154 @@
+import Joi from 'joi'
+
+/**
+ * This schema is used to validate the content for product landing pages.
+ * Validating our JSON content ensures that the content data type is accurate.
+ *
+ * The ProductLandingContent interface
+ * should match the Joi schema one-to-one.
+ * For this reason, the interface and schema are co-located here.
+ */
+
+/**
+ *
+ * Content Block Types & Schemas
```

Comment:
Have set up this schema for two main reasons:
1. Validation & type guarding, so we can better reason about data coming into the view
2. A form of docs for future iteration & maintenance (hopefully feels much clearer what the intended content structure is _with_ this schema vs without)